ipatests/test_ipaclient/test_cli_ipa_not_configured.py

```diff
@@ -0,0 +1,39 @@
+import subprocess
+
+import pytest
+
+from ipapython.admintool import SERVER_NOT_CONFIGURED
+
+
+class TestIPANotConfigured:
+    """
+    Test class for CLI commands with ipa server not configured.
+    """
+
+    @staticmethod
+    def run_command(args, return_code, expected_output, expected_error, unexpected_error=None):
+        """
+        Run command with ipa server not configured.
+        Launch the command specified in args.
+        Check that the exit code is as expected and that stdout and stderr
+        contain the expected strings.
+        """
+        p = subprocess.Popen(args, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        out, err = p.communicate()
+        assert return_code == p.returncode
+        if expected_output:
+            assert expected_output in out
+        if expected_error:
+            assert expected_error in err
+        if unexpected_error:
+            assert unexpected_error not in err
+
+    @pytest.mark.xfail(reason="Mismatch in return code, SERVER_NOT_CONFIGURED is 2, but ipa backup returns 1")
```

Comment:
No need for xfail as the command correctly returns 2 when IPA is not configured. You can test it for yourself:
```
[root ~]# ls /etc/ipa/default.conf
ls: cannot access '/etc/ipa/default.conf': No such file or directory
[root ~]# ipa-backup
IPA is not configured on this system.
The ipa-backup command failed.
[root ~]# echo $?
2
[root ~]# 
```

Reply:
okay, will correct that. ipa-backup indeed returns 1, but only if you are not root